schemas/capability.json

```diff
@@ -0,0 +1,691 @@
+        ]}
+      },
+      "required": ["angle", "blade"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["BladeSystemRotation"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "angle": { "$ref": "definitions.json#/entities/rotationAngle" }
+      },
+      "required": ["angle"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["FogOff"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {}
+      },
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["FogOn"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "fogOutput": { "$ref": "definitions.json#/entities/fogOutput" },
+        "fogType": { "enum": ["Fog", "Haze"] }
+      },
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["FogOutput"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "fogOutput": { "$ref": "definitions.json#/entities/fogOutput" }
+      },
+      "required": ["fogOutput"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["FogType"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "fogType": { "enum": ["Fog", "Haze"] }
+      },
+      "required": ["fogType"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["RotationAngle"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "angle": { "$ref": "definitions.json#/entities/rotationAngle" }
+      },
+      "required": ["angle"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["RotationSpeed"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "speed": { "$ref": "definitions.json#/entities/rotationSpeed" }
+      },
+      "required": ["speed"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["Speed"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "speed": { "$ref": "definitions.json#/entities/speed" }
+      },
+      "required": ["speed"],
+      "additionalProperties": false
+    },
+    {
+      "properties": {
+        "dmxRange": {},
+        "type": { "enum": ["Maintenance"] },
+        "comment": {},
+        "menuClick": {},
+        "switchChannels": {},
+        "parameter": { "$ref": "definitions.json#/entities/factor" },
```

Comment:
This may be a bit too restrictive, but we'll see in the future.

Reply:
It describes whether the capability is a step or a proportional range ...

Reply:
Done. (No action needed.)